package.json

```diff
@@ -59,11 +57,11 @@
 		"flat-zip": "^1.0.1",
 		"github-reserved-names": "^1.1.8",
 		"github-url-detection": "^1.1.5",
-		"image-promise": "^7.0.0",
+		"image-promise": "^7.0.1",
 		"indent-textarea": "^2.0.1",
 		"linkify-issues": "2.0.0-nolookbehind",
 		"linkify-urls": "3.1.0-nolookbehind",
-		"mem": "^6.0.1",
+		"mem": "^6.1.0",
```

Comment:
I actually had left this on 6.0.1 because it's technically a better version since we don't use `mem.clear`

Reply:
https://github.com/sindresorhus/refined-github/pull/3168/commits/3b998202f2cb574e196d275839062aa342b381a5